sdk/test-utils/recorder/karma.conf.js

```diff
@@ -110,7 +118,8 @@ module.exports = function(config) {
     browserDisconnectTimeout: 10000,
     browserDisconnectTolerance: 3,
     browserConsoleLogOptions: {
-      terminal: process.env.TEST_MODE !== "record"
+      // We would usually hide the logs from the tests, but we don't need to do this inside of the recorder package because we are not recording the tests.
+      // // terminal: process.env.TEST_MODE !== "record"
```

Comment:
should we just delete this code? or do we think folks will need to uncomment during debugging?

Reply:
I'm trying to prevent people from adding this code back. We have this code in most (if not all) of our karma.conf.js files. Adding this code silently, as in, in a way that other people might not be aware, or forget, will make it really hard to debug this specific library.